packages/cli/src/helpers/flags.ts

```diff
@@ -177,3 +178,17 @@ export const urlOrFrame = flags.build({
   default: () => getConfigDefault('urlOrFrame'),
   description: 'URL or Frame',
 });
+
+export const emulateDevice = flags.build({
+  char: 'e',
+  default: () => getConfigDefault('emulateDevice', defaultFlagArgs.emulateDevice),
+  description: 'Simulate a device\'s screen size.',
```

Comment:
swap to a template literal

Reply:
> Can we pull-in "simulate-device-options.json" from a 1st party source so we don't have to maintain this file as new devices roll out? Maybe from https://www.npmjs.com/package/chromedriver > http://chromedriver.chromium.org/mobile-emulation

I've tried adding "chromedriver" and "devtools-frontend" but upon global search of the snippets of that master list I found out that:

1. The chromedriver npm package is pretty bare minimal containing just a binary so the list of devices not in there
2. Although the list exists in the repo of [devtools-frontend](https://github.com/ChromeDevTools/devtools-frontend/blob/bd62b4d646a85a6b2f0860d3884c42fb5adc76d5/front_end/emulated_devices/module.json), the final built version in npm doesn't include it verbatim. I found this in node_modules that references the list https://paste.corp.linkedin.com/show/883950/ but it looks like nothing is exported in that file.

I think we may have to include the static file

Reply:
Ok that works for now then. I'll create a ticket for further discovery on this down the road.